AmericanMedicalAssociation/ama-style-guide-2
# EWL-8254: Explore topic section styling on sub-category pages

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- [Issue XXX: Issue Title](https://github.com/AmericanMedicalAssociation/AMA-style-guide/issues/XXX)

**Jira Ticket**
- [EWL-8254: Subcat | Explore Topics component](https://issues.ama-assn.org/browse/EWL-8254)

## Description
Added styling within subcategory_index for new explore topics component.


## To Test
Navigate to any subcategory page, confirm styling matches zeplins provided.

[Desktop](https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5f062b544417237955daff6e)

[Tablet](https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5f3fe827e829bd331bd42fdb)

[Mobile](https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5f3fe82b12485e157ba260ab)

## Visual Regressions

_Please provide the pull request ID below. This will then reference the automated VRT report after it has been generated._

A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/[PULL_REQUEST_ID]/html_report/index.html).

**Before proceeding:** Run visual regression tests locally to ensure new work does not introduce new bugs.

_The visual regression tests always compare against the production version of the style guide. Changes and new work will show up as test failures. Please describe your work so that the report can be reviewed by the team._

If you are creating or updating a pattern be sure you have created or updated the [scenario in `backstop.json`](ama-style-guide-2/styleguide/backstop.json).

_After changes in the report are documented and demonstrate your desired changes, please mark the pull request as `ready for review`.

For more information on visual regression testing review the [How do I run tests?](https://issues.ama-assn.org:8446/confluence/pages/viewpage.action?pageId=23298568) document in Confluence.


## Relevant Screenshots/GIFs
Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.


## Remaining Tasks
Things relevant to here may be updating [The Glossary](https://issues.ama-assn.org:8446/confluence/display/DTD/Glossary?src=contextnavpagetreemode) with the latest naming changes or review of the relative [D8](https://github.com/AmericanMedicalAssociation/ama-d8) work.


## Additional Notes
Anything more to add? Good things to call out here are:
- Contingent on [PR for EWL-2854](https://github.com/AmericanMedicalAssociation/ama-d8/pull/2284) being merged in, test with that branch checked out.

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
